### DISCUSSION TITLE
migrating from docker to LXC

### DISCUSSION THREAD

**Discussion by J0han3s** [photoprism/photoprism#4026] [Q&A]
Hi is anyone able to assist. 

I have installed photoprism lxc using https://tteck.github.io/Proxmox/ .

I have used bind mount to mount my originals and storage folder.  How do I also move my MySQL database folder over, for a quick clean migration? I can see the folder from docker, but not sure where to move it to and how to tell the LXC to use the data copied across

**Comment by graciousgrey** [comment]
You could create a backup on your old installation and restore it on your new setup.

https://docs.photoprism.app/user-guide/backups/
https://docs.photoprism.app/user-guide/backups/restore/
